packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec

```diff
@@ -1,66 +1,91 @@
+# template: smart_proxy_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global gem_name smart_proxy_ansible
+%global plugin_name ansible
 
+%global foreman_proxy_min_version 1.24
 %global foreman_proxy_dir /usr/share/foreman-proxy
-%global foreman_proxy_statedir %{_localstatedir}/lib/foreman-proxy
+%global foreman_proxy_statedir /var/lib/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
-%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
-%global smart_proxy_dynflow_bundlerd_dir %{?rhel:/opt/theforeman/tfm/root/}%{_datadir}/smart_proxy_dynflow_core/bundler.d
+%global foreman_proxy_settingsd_dir /etc/foreman-proxy/settings.d
+%global smart_proxy_dynflow_bundlerd_dir %{_datadir}/smart_proxy_dynflow_core/bundler.d
```

Comment:
Funny thing here: when building as non-scl, this still needs to put it in the SCL `bundler.d` file because it's actually for a *different* package. I didn't see it in the gem2rpm template file, but exactly the reason why we should clean this up. Did I mention this is an absolute mess?

It would make much more sense to have `tfm-rubygem(foreman_ansible_core)` contain this file and let that drop it in. This package would then depend on it with the version constraints it already has. Probably even better to have `tfm-rubygem(smart_proxy_dynflow_core)` depend on it since that really needs it.